shared/test/localization/format-test.js

```diff
@@ -187,7 +188,7 @@ describe('format', () => {
     });
 
     it('overwrites existing locale strings', async () => {
-      const filename = 'core/audits/is-on-https.js';
+      const filename = `${LH_ROOT}/core/audits/is-on-https.js`;
```

Comment:
seems like we might want to keep this for test coverage (maybe with a comment or a separate test for this case)

Reply:
done